fe/fe-core/src/main/java/org/apache/doris/clone/TabletScheduler.java

```diff
@@ -1725,6 +1739,7 @@ public boolean finishCloneTask(CloneTask cloneTask, TFinishTaskRequest request)
             LOG.warn("got unexpected exception when finish clone task. tablet: {}",
                     tabletCtx.getTabletId(), e);
             stat.counterTabletScheduledDiscard.incrementAndGet();
+            Env.getCurrentSystemInfo().getBackend(tabletCtx.getDestBackendId()).updateCloneFailedWindow();
```

Comment:
delete line 1742.  reason is like line 1729

Reply:
fixed